src/system/api/rqidApi.ts

```diff
@@ -114,12 +116,50 @@ export class Rqid {
       result.splice(0, 0, ...worldDocuments);
     }
 
+    const distinctWorldRqids: string[] = result
+      .map((d) => d.data?.flags?.rqg?.documentRqidFlags?.id)
+      .filter(function (value, index, self) {
+        return self.indexOf(value) === index;
+      });
```

Comment:
A more concise distinct might be
```suggestion
    const distinctWorldRqids: string[] = [
      ...new Set(result.map((d) => d.data?.flags?.rqg?.documentRqidFlags?.id)),
    ];
```
It creates a (distinct) Set from the array and then converts it back to an array.

Reply:
Changed to the recommended method for both world and compendium Rqids